src/auditwheel/repair.py

```diff
@@ -30,6 +30,18 @@
 ).match
 
 
+def _is_in_list(soname: str, items: Optional[List[str]]) -> Optional[str]:
+    for item in (items or []):
+        if item in soname:
```

Comment:
I think we should have an exact match. Why shouldn't we ?

Reply:
This could well be a lambda internal to `repair_wheel`:
```
_exclude_rule = lambda item: next((item in soname for item in exclude), None)
```
But I also wonder about partial string matching. We probably want very good control about whike libraries are excluded.
In case you're thinking about matching groups, then make it explicit, like mylibgroup* (check the glob module)